ImportBlocks/ImportBlocks.cs

```diff
@@ -48,8 +48,9 @@ private bool OnExport(string[] args)
                     if (start <= end)
                         fs.Write(BitConverter.GetBytes(count), 0, sizeof(uint));
                     fs.Seek(0, SeekOrigin.End);
-                    using (Snapshot snapshot = Blockchain.Singleton.GetSnapshot())
-                        for (uint i = start; i <= end; i++)
+                    for (uint i = start; i <= end; i++)
+                    {
+                        using (Snapshot snapshot = Blockchain.Singleton.GetSnapshot())
```

Comment:
I think that create new snapshot each time is not a good idea. We can use `Blockchain.Singleton.Store.GetBlock()`. It won't create any snapshot and read block from inner database directly.

Reply:
Maybe use the `store` directly in order to prevent the cache? or something for disable the cache of the objects in the snapshots?

Reply:
Yes, is better directly with the `Store`

Reply:
It won't increase the memory, because they will be released by GC. You can test it.

Reply:
But is stored in the internal `dictionary`, so we will have the same problem, we need 
```csharp
public TValue TryGet(TKey key, bool cache=true)
```

Reply:
So does `[]` operation,

Reply:
No. I'm **SURE** it will be released by GC automatically.

Reply:
https://github.com/neo-project/neo/blob/197a6ec216fe9ef2d17ac20f8ec6d95ed01fb1c3/neo/Persistence/Store.cs#L9

```c#
DataCache<UInt256, TrimmedBlock> IPersistence.Blocks => GetBlocks();
```

You can see the `=>`.

And https://github.com/neo-project/neo/blob/197a6ec216fe9ef2d17ac20f8ec6d95ed01fb1c3/neo/Persistence/LevelDB/LevelDBStore.cs#L44-L47

```c#
public override DataCache<UInt256, TrimmedBlock> GetBlocks()
{
    return new DbCache<UInt256, TrimmedBlock>(db, null, null, Prefixes.DATA_Block);
}
```

Reply:
I will  try and test.

Reply:
It will be disposed if we call `GetBlocks();` inside the for.